File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webbrowser.common.by import By


with webbrowser.Firefox() as browser:
	
    browser.get("https://training-support.net/webelements/dynamic-controls")
    print(f"Current page title: {browser.title}")

    checkbox = browser.find_element(By.XPATH,"//input[@id = 'checkbox']")
    print(f"Checkbox currently visible: {checkbox.is_displayed()}")

    toggle_button =  browser.find_element(By.XPATH,"//button[text() = 'Toggle Checkbox']")
    if(checkbox.is_displayed()) :
       toggle_button.click()
       print(f"Checkbox visible after toggle: {checkbox.is_displayed()}")
    browser.quit()

File: Selenium/Activities/Activity5.java
package examples;

import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class Activity5 {

	public static void main(String[] args) {

			WebDriver browser = new FirefoxDriver();
			browser.get("https://training-support.net/webelements/dynamic-controls");
			System.out.println("Current page title: "+ browser.getTitle());
			
			WebElement checkbox = browser.findElement(By.xpath("//input[@id = 'checkbox']"));
			System.out.println("Checkbox currently visible: " + checkbox.isDisplayed());
			
			WebElement toggleButton =  browser.findElement(By.xpath("//button[text() = 'Toggle Checkbox']"));
			
			if(checkbox.isDisplayed()) {
				toggleButton.click();
				System.out.println("Checkbox visible after toggle: " +checkbox.isDisplayed());
				
			}
			
			browser.quit();

	}

}
